site/content/docs/user/quick-start.md

```diff
@@ -148,14 +148,15 @@ kind delete cluster
 If the flag `--name` is not specified, kind will use the default cluster
 context name `kind` and delete that cluster.
 
-## Loading an Image Into Your Cluster
+## Loading Images Into Your Cluster
```

Comment:
This will break inbound heading links. The title change doesn't seem worth it in that light.

Reply:
Ahh, good point. Although I just found that there is only one inbound heading link which I guess we can simply change to point to `Loading images into your cluster` instead.
In my opinion, I guess we can go forwards with changing that inbound link as its a tiny change anyways but let me know what do you think :)

PS: [under this heading only is that one inbound heading link](https://github.com/kubernetes-sigs/kind/blob/master/site/content/docs/user/private-registries.md#pull-to-the-host-and-side-load)

Reply:
> Ahh, good point. Although I just found that there is only one inbound heading link which I guess we can simply change to point to Loading images into your cluster instead.

Er my concern is not the links within the site, but all the places we've handed out this link to users.

On most pages I'd agree, but this page has too much content (before we started making individual guides) and has been around a while, I'm certain there are many more links to this floating around out there 😅

Reply:
Coolz, sounds legit. I ll revert this one.